### PR TITLE
fix: fix style bug and sorting of widgets

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
commits!

### CODE REVIEW

File: apps/web/src/services/dashboards/widgets/_base/base-count-of-findings/BaseCountOfFindingsWidget.vue [hunk 1]
```diff
@@ -281,7 +281,7 @@ defineExpose<WidgetExpose<Data[]>>({
     .data-container {
         display: flex;
         flex-direction: column;
-        height: 100%;
+        height: 23.5rem;
```
Comment (yuda110) [inline apps/web/src/services/dashboards/widgets/_base/base-count-of-findings/BaseCountOfFindingsWidget.vue:284]: [[Major] 위젯 - CountOfFindings 두개 다 full mode 로 변경하면 데이터가 사라지고 위젯이 짧아짐](https://pyengine.atlassian.net/browse/QA-515)

File: apps/web/src/services/dashboards/widgets/asset-widgets/total-failure-and-severity/TotalFailureAndSeverityWidget.vue [hunk 2]
```diff
@@ -265,7 +265,7 @@ const fetchRealtimeData = async (): Promise<Data[]> => {
         apiQueryHelper
             .setFilters(state.consoleFilters)
             .addFilter({ k: 'ref_cloud_service_type.labels', v: 'Compliance', o: '=' })
-            .addFilter({ k: 'key', v: ['fail_finding_count', 'pass_findings_count'], o: '' });
+            .addFilter({ k: 'key', v: ['fail_finding_count', 'pass_finding_count'], o: '' });
```
Comment (yuda110) [inline apps/web/src/services/dashboards/widgets/asset-widgets/total-failure-and-severity/TotalFailureAndSeverityWidget.vue:268]: [[Major] 위젯 - Total Failure and Severity 그래프에 Failure Rate 데이터 확인 필요 ](https://pyengine.atlassian.net/browse/QA-527)

Comment (yuda110) [reply apps/web/src/services/dashboards/widgets/asset-widgets/total-failure-and-severity/TotalFailureAndSeverityWidget.vue:268]: ~~copilot's~~my mistake

File: apps/web/src/services/dashboards/widgets/asset-widgets/trend-of-pass-and-fail-findings/TrendOfPassAndFailFindingsWidget.vue [hunk 3]
```diff
@@ -1,6 +1,7 @@
 <template>
     <widget-frame v-bind="widgetFrameProps"
                   class="trend-of-pass-and-fail-findings"
+                  refresh-on-resize
```
Comment (yuda110) [inline apps/web/src/services/dashboards/widgets/asset-widgets/trend-of-pass-and-fail-findings/TrendOfPassAndFailFindingsWidget.vue:4]: [[Major] 위젯 - TrendOfPassAndFailFindings 그래프와 아래 차트 내용이 일치하지 않음](https://pyengine.atlassian.net/browse/QA-537)

스크린샷 2023-06-15 오후 1 28 04

File: apps/web/src/services/dashboards/widgets/_base/base-count-of-findings/BaseCountOfFindingsWidget.vue (hunk 1 above)
Comment (WANZARGEN) [reply apps/web/src/services/dashboards/widgets/_base/base-count-of-findings/BaseCountOfFindingsWidget.vue:284]: question: is this widget's height always the same?

File: apps/web/src/services/dashboards/widgets/asset-widgets/total-failure-and-severity/TotalFailureAndSeverityWidget.vue [hunk 4]
```diff
@@ -313,6 +316,11 @@ const drawChart = (chartData: XYChartData[]) => {
         dateFormat: DATE_FORMAT,
         dateFields: [DATE_FIELD_NAME],
     });
+
+    const tooltip = chartHelper.createTooltip();
+    chartHelper.setXYSingleTooltipText(chart, tooltip);
+    series.set('tooltip', tooltip);
+
```
Comment (yuda110) [inline apps/web/src/services/dashboards/widgets/asset-widgets/total-failure-and-severity/TotalFailureAndSeverityWidget.vue:323]: [[Minor] 위젯 - TotalFailureAndSeverity 그래프에 툴팁 추가](https://pyengine.atlassian.net/browse/QA-529)

스크린샷 2023-06-15 오후 1 33 14

File: apps/web/src/services/dashboards/widgets/asset-widgets/compliance-check-status/ComplianceCheckStatusWidget.vue [hunk 5]
```diff
@@ -20,7 +20,7 @@
                             {{ $t('DASHBOARDS.WIDGET.COMPLIANCE_CHECK_STATUS.TOTAL_COMPLIANCE_NUMBER') }}
                         </p>
                         <p class="value">
-                            {{ state.complianceCount === undefined ? '--' : state.complianceCount }}
+                            {{ state.complianceCount === undefined ? '--' : commaFormatter(state.complianceCount) }}
```
Comment (WANZARGEN) [inline apps/web/src/services/dashboards/widgets/asset-widgets/compliance-check-status/ComplianceCheckStatusWidget.vue:23]: praise: good! `commaFormatter` :)

**Review by skdud4659 — APPROVED** [review]
SSG! LGTM!!!

File: apps/web/src/services/dashboards/widgets/asset-widgets/trend-of-pass-and-fail-findings/TrendOfPassAndFailFindingsWidget.vue (hunk 3 above)
Comment (WANZARGEN) [reply apps/web/src/services/dashboards/widgets/asset-widgets/trend-of-pass-and-fail-findings/TrendOfPassAndFailFindingsWidget.vue:4]: praise: simple solution

**Review by WANZARGEN — APPROVED** [review]
LGTM!

File: apps/web/src/services/dashboards/widgets/_base/base-count-of-findings/BaseCountOfFindingsWidget.vue (hunk 1 above)
Comment (yuda110) [reply apps/web/src/services/dashboards/widgets/_base/base-count-of-findings/BaseCountOfFindingsWidget.vue:284]: In widgets, min-height is 400px(25rem) and max-height is 560px(35rem). But there's some style issue with WidgetFrame, so I set static height to widget

Comment (yuda110) [reply apps/web/src/services/dashboards/widgets/_base/base-count-of-findings/BaseCountOfFindingsWidget.vue]: update code to set height only full mode

**Pull request merged**